### PR TITLE
add a registry auth tutorial

### PR DESCRIPTION
**Pull request by mikebrow**
Addressing at least a portion of #1482

Needed some documentation for _json_key authentication with a registry.

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: docs/registry.md [hunk 1]
```diff
@@ -86,3 +86,56 @@ The registry credential in this config will only be used when auth config is
...
 
 After modify this config, you need restart the `containerd` service.
+
+### Configure Registry Credentials Example - GCR with _json_key Authentication
+
+Create a gcp account with gcr, do all the steps to enable receiving a
+pushed image for a gcr instance, including the generation and download of a
+new _json_key (for a new service account user.) Then:
+
+```bash
```
Comment (dmcgowan) [inline docs/registry.md:96]: What is this section necessary for?

Comment (mikebrow) [reply docs/registry.md:99]: fair question... updating.

**Pull request merged**